### DISCUSSION TITLE
BOINC Integration

### DISCUSSION THREAD

**Discussion by AenBleidd** [LeelaChessZero/lc0#1951] [Ideas]
Hello folks,

Is there any interest of bringing this project to the BOINC infrastructure that is running on thousands of volunteer devices?
You might heard about some of the popular projects like SETI@home, World Community Grid, Einstein@home, etc.
You can check the details here: https://boinc.berkeley.edu/

If you would like to explore this opportunity, I'll be happy to make an introduction to it.

Don't hesitate to contact either me via email ([lestat.de.lionkur@gmail.com](mailto:lestat.de.lionkur@gmail.com)) or the Director of BOINC David P. Anderson: https://boinc.berkeley.edu/anderson/

**Comment by mooskagh** [comment]
Sorry for not responding earlier!  
It certainly sounds interesting, just noone seems to have time for this project right now.  
So this response is to tell you that your message is seen, hopefully someone from the Lc0 team will follow up on it.
